sdk/src/cameras/itof-camera/camera_itof.h

```diff
@@ -139,6 +139,8 @@ class CameraItof : public aditof::Camera {
     aditof::Status getTemperatureSensors(
         std::vector<std::shared_ptr<aditof::TemperatureSensorInterface>>
             &sensors) override;
+    aditof::Status CameraItof::adsd3500_set_toggle_mode(int mode);
```

Comment:
You need to remove CameraItof:: from these 2 lines, linux build failes with them here.

Reply:
done